applications/nrf_desktop/configuration/nrf54l15pdk_nrf54l15_cpuapp/app.overlay

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* Redefine the partition map. */
+&rram0 {
+	/delete-node/ partitions;
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+		app_partition: partition@0 {
+			label = "application";
+			reg = <0x0 0x177000>;
+		};
+
+		storage_partition: partition@177000 {
+			label = "storage";
+			reg = <0x177000 0x6000>;
+		};
+	};
+};
```

Comment:
To be considered: Maybe we could wait for your PM fix (https://github.com/nrfconnect/sdk-nrf/pull/14368) and replace this DTS partition definitions with pm_static (if necessary)?

Reply:
To be honest, I would merge it as is and then switch to Partition Manager together with MCUboot integration :)

Reply:
Alright, if no one will object then let's leave it for now as it is.